data/2021-02-27_penn_invitational_c.yaml

```diff
@@ -51,15 +51,19 @@ Teams:
   state: PA
 - number: 3
   school: Athens Area High School
+  suffix: A
   state: PA
 - number: 4
   school: Athens Area High School
+  suffix: B
   state: PA
 - number: 5
   school: Bayard Rustin High School
+  suffix: Rustin Blue
```

Comment:
We don't need the school name in the suffix (otherwise it'll say "Bayard Rustin H.S. Rustin Blue"), so simply "Blue" should suffice.